public/locales/es/common.json

```
@@ -129,6 +132,18 @@
+  "volunteer": "Voluntario",
+  "moreCount": "+{{count}} más",
+  "togglePledgedRaised": "Alternar entre cantidades prometidas y recaudadas",
+  "accept": "Aceptar",
+  "acceptedSuccessfully": "Aceptado con éxito",
+  "decline": "Rechazar",
+  "declinedSuccessfully": "Rechazado con éxito",
+  "reject": "Rechazar",
+  "rejectedSuccessfully": "Rechazado con éxito",
+  "searchRequests": "Buscar solicitudes",
+  "title": "Título",
+  "assignee": "Asignado"
```

Biome (2.1.2)

[error] The key title was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)